### DISCUSSION TITLE
Question about R Studio Replacement with Positron

### DISCUSSION THREAD

**Discussion by ghost** [dat-linux/community#65] [General]
irst of all, thank you for all the hard work you do on this amazing project!

I have recently come across information about a new R IDE called **Positron** that is currently under development. I am curious if there are any plans to replace R Studio with Positron in the near future?

If such a transition is planned, could you provide some insight into the timeline and any major changes we can expect? Any details about the new features or improvements that Positron will bring would also be greatly appreciated.

Looking forward to your response!

**Comment by dat-linux** [comment]
Hi,

We'll be adding Positron as a DAT Linux app in the near future, depending on stability. But RStudio will continue to be available as well. It won't be a replacement, but a choice.

Sent with [Proton Mail](https://proton.me/) secure email.
